code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -463,5 +468,35 @@
 	piping_layer = 4
 	icon_state = "scrub_map_on-4"
 
+/obj/machinery/atmospherics/components/unary/vent_scrubber/plunger_act(obj/item/plunger/P, mob/living/user, reinforced)
```

Comment:
```suggestion
/obj/machinery/atmospherics/components/unary/vent_scrubber/plunger_act(obj/item/plunger/plunger, mob/living/user, reinforced)
```
Better var names. Don't need to change all the other implementations, just this one is fine.